### DISCUSSION TITLE
Release a google forms locked mode bypass?

### DISCUSSION THREAD

**Discussion by Maxxy1231sad** [3kh0/ext-remover#859] [Polls JOIN TITANIUM NETWORK TO SHARE PLEASE]
1. yes its possble
2. It will be obfuscated and paid for.
Why? Cuz money is money u gotta get by somehow
Can I share the key I buy?
no :)
why is t gonna be obfuscated>
skids and it will get patched MUCH quicker if I dont and im trying to get out of school before that happens lol

**Comment by rwg100** [comment]
wtf are you even saying dawg

**Reply by Maxxy1231sad** [reply]
im saying im one of the only ppl who has a bypass for google forms locked mode :p

**Reply by AshtonDavies** [reply]
SpaceSaver2000 found one in TN.

**Comment by funkymokey** [comment]
skid account wanting to sound like a cool programmer :0 (literally me)

**Comment by Epicminer256** [comment]
Bro you can just modify the form site itself

https://github.com/cheatsoft/Google-Forms-Unlocked

**Reply by Epicminer256** [reply]
Respect the hustle, but honestly we are all a bunch of broke students. And even then, someone is going to try and make it from scratch for free before you sell it.

**Reply by AshtonDavies** [reply]
Fax

**Reply by Epicminer256** [reply]
Your paid code may just not be worth it. Code that is not easy to maintain and there was no thought into the announcement just doesn't make me think it wouldn't be worth it. We could just get sold very broken code with a broken "key" system that can lock us out of the code.

We could make a free version that works better then this paid version.

**Reply by Maxxy1231sad** [reply]
Fair lol

**Comment by xNasuni** [comment]
https://github.com/xNasuni/google-forms-unlocker

**Comment by lolpc100** [comment]
> 1. yes its possble
> 2. It will be obfuscated and paid for.
>    Why? Cuz money is money u gotta get by somehow
>    Can I share the key I buy?
>    no :)
>    why is t gonna be obfuscated>
>    skids and it will get patched MUCH quicker if I dont and im trying to get out of school before that happens lol
